data/AdityaChaudhary3.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Aditya Chaudhary",
+  "type": "personal",
+  "bio": "Web Developer | DevOps | Open Source",
+  
+  "tags": ["Web Developer", "Open Source", "DevOps"],
+   "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/aditya_script"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://twitter.com/aditya_script"
```

Comment:
I think you want to use your **" Linkedin "** url here :+1: 
```Suggestion
      "url": "https://www.linkedin.com/in/aditya-chaudhary3/"
```

Reply:
yeah, Thanks for pointing out this.